cc/engine/templates/macros_templates/deed.html

```diff
@@ -182,19 +182,11 @@ <h2>
     {% endblock %}
   </div>
 
-  <script type='text/javascript' src='/wp-content/themes/cc/js/cc.js'></script>
-  <script type='text/javascript' src='/wp-content/plugins/cc-donate/js/cc-donate.js'></script>
-  <script type='text/javascript' src='/wp-content/themes/cc/js/breakpoint-body-class.js'></script>
-  <script type='text/javascript' src='/wp-content/themes/cc/js/sticky-nav.js'></script>
-  <script type='text/javascript' src='/wp-content/themes/cc/js/toggle-search.js'></script>
-  <script type='text/javascript' src='/wp-content/themes/cc/js/donation.js'></script>
-  <script type='text/javascript' src='/wp-content/themes/twentysixteen/js/skip-link-focus-fix.js'></script>
   <script type='text/javascript'>
   /* <![CDATA[ */
   var screenReaderText = {"expand":"{% trans %}expand child menu{% endtrans %}","collapse":"{% trans %}collapse child menu{% endtrans %}"};
   /* ]]> */
   </script>
-  <script type='text/javascript' src='/wp-content/themes/twentysixteen/js/functions.js'></script>
```

Comment:
this is the script that handles the menu toggle 
![Screen Shot 2020-09-16 at 3 07 31 PM](https://user-images.githubusercontent.com/894708/93375351-6b43a200-f82e-11ea-9c21-b1f794e26a33.png)